examples/full/RegistrationForm/validation-rules.js

```diff
@@ -1,6 +1,21 @@
 import isEmail from 'validator/lib/isEmail'
 
 export default {
+  fieldPath: {
```

Comment:
Inclusion of `type` and `name` intermediate selectors break the concept of `fieldPath`. 
Effectively, this is better off as `fieldGroup`.

Reply:
**Update:** Adopted the `fieldGroup` key name.